pkg/subctl/cmd/gather/resource.go

```diff
@@ -26,22 +26,27 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-func ResourcesToYAMLFile(info *Info, ofType schema.GroupVersionResource, namespace string) {
+func ResourcesToYAMLFile(info *Info, ofType schema.GroupVersionResource, namespace string, listOptions metav1.ListOptions) {
 	err := func() error {
-		list, err := info.DynClient.Resource(ofType).Namespace(namespace).List(metav1.ListOptions{})
+		list, err := info.DynClient.Resource(ofType).Namespace(namespace).List(listOptions)
 		if err != nil {
 			return errors.WithMessagef(err, "error listing %q", ofType.Resource)
 		}
 
-		info.Status.QueueSuccessMessage(fmt.Sprintf("Found %d %s in namespace %q", len(list.Items), ofType.Resource, namespace))
+		selectorStr := ""
+		if listOptions.LabelSelector != "" {
```

Comment:
Do we want to show this to user? Feels like exposing too much of internal implementation than needed. But can see value in troubleshooting gather command, so okay either ways.

Reply:
The reason I did this was to provide more context as to what's being gathered  when searching by selector. Otherwise you see:

 ```
✓ Found 1 daemonsets in namespace "submariner-operator"
 ✓ Found 1 daemonsets in namespace "submariner-operator"
 ✓ Found 1 daemonsets in namespace "submariner-operator"
```

so you can't tell what's actually being collected. Now you see:

```
 ✓ Found 1 daemonsets by label selector "app=submariner-gateway" in namespace "submariner-operator"
 ✓ Found 1 daemonsets by label selector "app=submariner-routeagent" in namespace "submariner-operator"
 ✓ Found 1 daemonsets by label selector "app=submariner-globalnet" in namespace "submariner-operator"
```